appveyor.yml

```diff
@@ -140,6 +140,8 @@ test_script:
   - if x%USE_PYTEST% == xno python tests.py %PYTEST_ARGS%
   # Generate a html for visual tests
   - python visual_tests.py
+  - if x%USE_PYTEST% == xyes pip install codecov
+  - if x%USE_PYTEST% == xyes codecov -e PYTHON_VERSION PLATFORM
```

Comment:
These lines mean that codecov only happens if `USE_PYTEST` is true, which is only for the one build. pytest is still being run in the other builds, just through `tests.py` in the root which calls `test()` in `matplotlib/__init__.py`.